extensions/wikia/AdEngine/js/video/player/porvata/googleImaPlayerFactory.js

```diff
@@ -107,7 +114,11 @@ define('ext.wikia.adEngine.video.player.porvata.googleImaPlayerFactory', [
 		}
 
 		function dispatchEvent(eventName) {
-			adsManager.dispatchEvent(eventName);
+			if (eventListeners[eventName] && eventListeners[eventName].length > 0) {
+				eventListeners[eventName].forEach(function (callback) {
+					callback({});
```

Comment:
Because of: https://github.com/Wikia/app/blob/dev/extensions/wikia/AdEngine/js/video/player/porvata/porvataTracker.js#L65